Mathlib/Algebra/BigOperators/Group/Finset.lean

```diff
@@ -539,15 +539,18 @@ theorem prod_biUnion [DecidableEq α] {s : Finset γ} {t : γ → Finset α}
     (hs : Set.PairwiseDisjoint (↑s) t) : ∏ x ∈ s.biUnion t, f x = ∏ x ∈ s, ∏ i ∈ t x, f i := by
   rw [← disjiUnion_eq_biUnion _ _ hs, prod_disjiUnion]
 
-/-- Product over a sigma type equals the product of fiberwise products. For rewriting
+/-- Product over a sigma type equals the product of the fiberwise products. For rewriting
```

Comment:
This change looks negative to me - the earlier version reads cleaner. Alternatively,
"The product over the sigma type equals the product of the fiberwise products." would be okay too. Similarly below.

Reply:
I think I like your second version better